drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -151,6 +148,7 @@ def items_without_answers():
             for item in items:
                 del item['feedback']
                 del item['zone']
+                item['displayName'] = self.ugettext(item['displayName'])
```

Comment:
@itsjeyd If we have translations msgstr in .po files with for HTML markup then `self.ugettext` will perform the translation for those, otherwise it will be render as it is. 
Currently, I was seeing that, item displayName has the following values:

- "Goes to the top"
- "Goes to the middle"
- "Goes to the bottom"
- "I don't belong anywhere"

correct me If I am missing . 
Thanks